### ISSUE TITLE
QubesOS R3.2 installment issue

### ISSUE THREAD

**Issue by TheProtocoler** [QubesOS/qubes-issues#4012]
### Qubes OS version:

The version I'm running is QubesOS **R3.2**

### Affected component(s): 

QubesOS doesn't recognize that I installed it. Whenever I install QubesOS, shut down and restart, it will give me this same screen: 

![image](https://user-images.githubusercontent.com/21243470/41501667-0ac335d4-71a9-11e8-8560-d840bd333d5b.png)

---

### Steps to reproduce the behavior:

I usually go through the normal installment method. 
1. "Install Qubes R3.2"
2. Choosing the language, keyboard lay-out
3.  Choosing my installation destination (I just double check, i don't change anything inside it except that I untick "Encrypt my data").

![image](https://user-images.githubusercontent.com/21243470/41501692-803df6aa-71a9-11e8-8ae8-97ed4c649f87.png)

One thing to note in that picture is that it has 1.97MiB of free space. I had 40GBs assigned to that partition before I powered it on again (so my previous install did happen and it did take up my space).

If I click on _RECLAIM DISK SPACE_ this is the information:

![image](https://user-images.githubusercontent.com/21243470/41501715-d4c4f9a8-71a9-11e8-8236-6c5654a86701.png)

So it did install, however, when restarting it just goes to this same screen:

![image](https://user-images.githubusercontent.com/21243470/41501667-0ac335d4-71a9-11e8-8560-d840bd333d5b.png)

Thank you.

**Comment by andrewdavidwong** [comment]
Did you forget to remove the installation medium (e.g., USB drive or DVD) and/or change the device boot priority after installing?

**Comment by TheProtocoler** [comment]
@andrewdavidwong 
There are no installation mediums in this process. As a matter of fact, I'm using VirtualBox.

**Comment by TheProtocoler** [comment]
@andrewdavidwong I just removed it and got it working! Thank you. I never removed my installation ISO on my other VM's. Will be a good habit from now on.

I'll list down the steps for users experiencing the same problem:
1. Detach the ISO.
2. You will be redirected into a terminal saying that it can't find the OS or something like that. Just type _exit_
3. After that, you'll probably be greeted by a screen like this:
![image](https://user-images.githubusercontent.com/21243470/41501886-15a371b8-71ad-11e8-9523-6625bcc23990.png)
Since it won't start up correctly I figured QubesOS *must* be installed, just not recognized for start up

4. So I went to *_Boot Maintenance Manager_* 

![image](https://user-images.githubusercontent.com/21243470/41501908-7e1b6cbe-71ad-11e8-88b0-e1db6cf1478d.png)

5. *_Boot From File_* and then choose your _volume label_ and explore to your Qubes EFI:

![image](https://user-images.githubusercontent.com/21243470/41501916-a7387c90-71ad-11e8-8efe-f49450e5f0d7.png)

There we go!
